addons/mail/static/src/components/composer/composer.scss

```diff
@@ -259,3 +259,9 @@
 .o_Composer_toolButtonSeparator {
     border-left: $border-width solid $border-color;
 }
+
+.o_ComposerSuggestion.dropdown-item {
+    &.active, &:active {
+        background-color: $gray-300;
+    }
+}
```

Comment:
Instead of this, change template of `mail.ComposerSuggestion` so that this:
```
t-att-class="{ 'active': props.isActive }"
```
becomes this:
```
t-att-class="{ 'active bg-300': props.isActive }"
```

This will lack `:active`, so I don't mind having SCSS just for that.
But put in in proper scss file, namely: `composer_suggestion.scss`:
```scss
.o_ComposerSuggestion {
   &:active {
      background-color: $gray-300;
   }
}
```

Reply:
@alexkuhn Done!